Source/WebKit/UIProcess/Extensions/WebExtension.h

```diff
@@ -245,6 +245,9 @@ class WebExtension : public API::ObjectImpl<API::Object::Type::WebExtension>, pu
     NSString *backgroundContentPath();
     NSString *generatedBackgroundContent();
 
+    bool hasInspectorBackgroundPage();
+    NSString *inspectorBackgroundPagePath();
```

Comment:
Shouldn't these declarations (and the others you're adding in this header) be conditionalized on `ENABLE(INSPECTOR_EXTENSIONS)`? I see you've made some of this implementations unconditional, but I'm not sure that's the right thing to do.

Reply:
These were intentional, so even on iOS we can check. This will likely we exposed on the API object too, and it does not make sense to allow checking it to only work on macOS. (We might just have it always return NO on iOS.)